sklearn/notebooks/hp_tuning_auto_mpg/ml_engine_hp_tuning_training.ipynb

```diff
@@ -0,0 +1,907 @@
+    "* `python-version` - The Python version to use for the job. Python 3.5 is available with runtime version 1.4 or greater. If you don't specify a Python version, the training service uses Python 2.7.\n",
+    "* `scale-tier` - A scale tier specifying the type of processing cluster to run your job on. This can be the CUSTOM scale tier, in which case you also explicitly specify the number and type of machines to use.\n",
+    "* `config` - Path to the job configuration file. This file should be a YAML document (JSON also accepted) containing a Job resource as defined in the API\n",
+    "\n",
+    "Note: Check to make sure gcloud is set to the current PROJECT_ID"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\u001b[1;31mERROR:\u001b[0m (gcloud.config.set) argument VALUE: Must be specified.\r\n",
+      "Usage: gcloud config set SECTION/PROPERTY VALUE [optional flags]\r\n",
+      "  optional flags may be  --help | --installation\r\n",
+      "\r\n",
+      "For detailed information on this command and its flags, run:\r\n",
+      "  gcloud config set --help\r\n"
+     ]
+    }
+   ],
+   "source": [
+    "! gcloud config set project $PROJECT_ID"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Submit the training job."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Job [auto_mpg_hp_tuning_20180912_103335] submitted successfully.\r\n",
+      "Your job is still active. You may view the status of your job with the command\r\n",
+      "\r\n",
+      "  $ gcloud ml-engine jobs describe auto_mpg_hp_tuning_20180912_103335\r\n",
+      "\r\n",
+      "or continue streaming the logs with the command\r\n",
+      "\r\n",
+      "  $ gcloud ml-engine jobs stream-logs auto_mpg_hp_tuning_20180912_103335\r\n",
+      "jobId: auto_mpg_hp_tuning_20180912_103335\r\n",
+      "state: QUEUED\r\n"
+     ]
+    }
+   ],
+   "source": [
+    "! gcloud ml-engine jobs submit training auto_mpg_hp_tuning_$(date +\"%Y%m%d_%H%M%S\") \\\n",
+    "  --job-dir $JOB_DIR \\\n",
+    "  --package-path $TRAINER_PACKAGE_PATH \\\n",
+    "  --module-name $MAIN_TRAINER_MODULE \\\n",
+    "  --region $REGION \\\n",
+    "  --runtime-version=$RUNTIME_VERSION \\\n",
+    "  --python-version=$PYTHON_VERSION \\\n",
+    "  --scale-tier BASIC \\\n",
+    "  --config $HPTUNING_CONFIG"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# [Optional] StackDriver Logging\n",
+    "You can view the logs for your training job:\n",
+    "1. Go to https://console.cloud.google.com/\n",
+    "1. Select \"Logging\" in left-hand pane\n",
+    "1. Select \"Cloud ML Job\" resource from the drop-down\n",
```

Comment:
I initially thought there is an option called "Cloud ML Jobs" in the left pane.  So, it confused me a little. I think Step #3 should be rephrased a little to something like this:

Go to "ML Engine" and select Jobs.

Reply:
Fixed